debian package build fails on stock Ubuntu 14.04

`sudo pbuilder build ../walinuxagent_1.3-0ubuntu1.dsc` fails with:

    W: /home/ubuntu/.pbuilderrc does not exist
    I: using fakeroot in build.
    I: Current time: Tue Aug 23 18:56:45 UTC 2016
    I: pbuilder-time-stamp: 1471978605
    I: Building the build Environment
    I: extracting base tarball [/var/cache/pbuilder/base.tgz]
    I: creating local configuration
    I: copying local configuration
    I: mounting /proc filesystem
    I: mounting /run/shm filesystem
    I: mounting /dev/pts filesystem
    I: policy-rc.d already exists
    I: Obtaining the cached apt archive contents
    I: Installing the build-deps
     -> Attempting to satisfy build-dependencies
     -> Creating pbuilder-satisfydepends-dummy package
    Package: pbuilder-satisfydepends-dummy
    Version: 0.invalid.0
    Architecture: amd64
    Maintainer: Debian Pbuilder Team <pbuilder-maint@lists.alioth.debian.org>
    Description: Dummy package to satisfy dependencies with aptitude - created by pbuilder
     This package was created automatically by pbuilder to satisfy the
     build-dependencies of the package being currently built.
    Depends: debhelper (>= 8), python-all
    dpkg-deb: building package `pbuilder-satisfydepends-dummy' in `/tmp/satisfydepends-aptitude/pbuilder-satisfydepends-dummy.deb'.
    Selecting previously unselected package pbuilder-satisfydepends-dummy.
    (Reading database ... 11859 files and directories currently installed.)
    Preparing to unpack .../pbuilder-satisfydepends-dummy.deb ...
    Unpacking pbuilder-satisfydepends-dummy (0.invalid.0) ...
    dpkg: pbuilder-satisfydepends-dummy: dependency problems, but configuring anyway as you requested:
     pbuilder-satisfydepends-dummy depends on debhelper (>= 8); however:
      Package debhelper is not installed.
     pbuilder-satisfydepends-dummy depends on python-all; however:
      Package python-all is not installed.
    
    Setting up pbuilder-satisfydepends-dummy (0.invalid.0) ...
    Reading package lists...
    Building dependency tree...
    Reading state information...
    Initializing package states...
    Writing extended state information...
    The following NEW packages will be installed:
      bsdmainutils{a} debhelper{a} dh-apparmor{a} file{a} gettext{a} 
      gettext-base{a} groff-base{a} intltool-debian{a} libasprintf0c2{a} 
      libcroco3{a} libexpat1{a} libffi6{a} libglib2.0-0{a} libmagic1{a} 
      libpipeline1{a} libpython-stdlib{a} libpython2.7-minimal{a} 
      libpython2.7-stdlib{a} libssl1.0.0{a} libunistring0{a} libxml2{a} 
      man-db{a} mime-support{a} po-debconf{a} python{a} python-all{a} 
      python-minimal{a} python2.7{a} python2.7-minimal{a} 
    0 packages upgraded, 29 newly installed, 0 to remove and 0 not upgraded.
    Need to get 0 B/10.7 MB of archives. After unpacking 41.9 MB will be used.
    Writing extended state information...
    debconf: delaying package configuration, since apt-utils is not installed
    Selecting previously unselected package libexpat1:amd64.
    (Reading database ... 11859 files and directories currently installed.)
    Preparing to unpack .../libexpat1_2.1.0-4ubuntu1_amd64.deb ...
    Unpacking libexpat1:amd64 (2.1.0-4ubuntu1) ...
    Selecting previously unselected package libffi6:amd64.
    Preparing to unpack .../libffi6_3.1~rc1+r3.0.13-12_amd64.deb ...
    Unpacking libffi6:amd64 (3.1~rc1+r3.0.13-12) ...
    Selecting previously unselected package libmagic1:amd64.
    Preparing to unpack .../libmagic1_1%3a5.14-2ubuntu3_amd64.deb ...
    Unpacking libmagic1:amd64 (1:5.14-2ubuntu3) ...
    Selecting previously unselected package libssl1.0.0:amd64.
    Preparing to unpack .../libssl1.0.0_1.0.1f-1ubuntu2_amd64.deb ...
    Unpacking libssl1.0.0:amd64 (1.0.1f-1ubuntu2) ...
    Selecting previously unselected package libasprintf0c2:amd64.
    Preparing to unpack .../libasprintf0c2_0.18.3.1-1ubuntu2_amd64.deb ...
    Unpacking libasprintf0c2:amd64 (0.18.3.1-1ubuntu2) ...
    Selecting previously unselected package libglib2.0-0:amd64.
    Preparing to unpack .../libglib2.0-0_2.40.0-2_amd64.deb ...
    Unpacking libglib2.0-0:amd64 (2.40.0-2) ...
    Selecting previously unselected package libpipeline1:amd64.
    Preparing to unpack .../libpipeline1_1.3.0-1_amd64.deb ...
    Unpacking libpipeline1:amd64 (1.3.0-1) ...
    Selecting previously unselected package libxml2:amd64.
    Preparing to unpack .../libxml2_2.9.1+dfsg1-3ubuntu4_amd64.deb ...
    Unpacking libxml2:amd64 (2.9.1+dfsg1-3ubuntu4) ...
    Selecting previously unselected package groff-base.
    Preparing to unpack .../groff-base_1.22.2-5_amd64.deb ...
    Unpacking groff-base (1.22.2-5) ...
    Selecting previously unselected package bsdmainutils.
    Preparing to unpack .../bsdmainutils_9.0.5ubuntu1_amd64.deb ...
    Unpacking bsdmainutils (9.0.5ubuntu1) ...
    Selecting previously unselected package man-db.
    Preparing to unpack .../man-db_2.6.7.1-1_amd64.deb ...
    Unpacking man-db (2.6.7.1-1) ...
    Selecting previously unselected package libcroco3:amd64.
    Preparing to unpack .../libcroco3_0.6.8-2ubuntu1_amd64.deb ...
    Unpacking libcroco3:amd64 (0.6.8-2ubuntu1) ...
    Selecting previously unselected package libunistring0:amd64.
    Preparing to unpack .../libunistring0_0.9.3-5ubuntu3_amd64.deb ...
    Unpacking libunistring0:amd64 (0.9.3-5ubuntu3) ...
    Selecting previously unselected package libpython2.7-minimal:amd64.
    Preparing to unpack .../libpython2.7-minimal_2.7.6-8_amd64.deb ...
    Unpacking libpython2.7-minimal:amd64 (2.7.6-8) ...
    Selecting previously unselected package python2.7-minimal.
    Preparing to unpack .../python2.7-minimal_2.7.6-8_amd64.deb ...
    Unpacking python2.7-minimal (2.7.6-8) ...
    Selecting previously unselected package file.
    Preparing to unpack .../file_1%3a5.14-2ubuntu3_amd64.deb ...
    Unpacking file (1:5.14-2ubuntu3) ...
    Selecting previously unselected package mime-support.
    Preparing to unpack .../mime-support_3.54ubuntu1_all.deb ...
    Unpacking mime-support (3.54ubuntu1) ...
    Selecting previously unselected package gettext-base.
    Preparing to unpack .../gettext-base_0.18.3.1-1ubuntu2_amd64.deb ...
    Unpacking gettext-base (0.18.3.1-1ubuntu2) ...
    Selecting previously unselected package gettext.
    Preparing to unpack .../gettext_0.18.3.1-1ubuntu2_amd64.deb ...
    Unpacking gettext (0.18.3.1-1ubuntu2) ...
    Selecting previously unselected package intltool-debian.
    Preparing to unpack .../intltool-debian_0.35.0+20060710.1_all.deb ...
    Unpacking intltool-debian (0.35.0+20060710.1) ...
    Selecting previously unselected package po-debconf.
    Preparing to unpack .../po-debconf_1.0.16+nmu2ubuntu1_all.deb ...
    Unpacking po-debconf (1.0.16+nmu2ubuntu1) ...
    Selecting previously unselected package dh-apparmor.
    Preparing to unpack .../dh-apparmor_2.8.95~2430-0ubuntu5_all.deb ...
    Unpacking dh-apparmor (2.8.95~2430-0ubuntu5) ...
    Selecting previously unselected package debhelper.
    Preparing to unpack .../debhelper_9.20131227ubuntu1_all.deb ...
    Unpacking debhelper (9.20131227ubuntu1) ...
    Selecting previously unselected package libpython2.7-stdlib:amd64.
    Preparing to unpack .../libpython2.7-stdlib_2.7.6-8_amd64.deb ...
    Unpacking libpython2.7-stdlib:amd64 (2.7.6-8) ...
    Selecting previously unselected package libpython-stdlib:amd64.
    Preparing to unpack .../libpython-stdlib_2.7.5-5ubuntu3_amd64.deb ...
    Unpacking libpython-stdlib:amd64 (2.7.5-5ubuntu3) ...
    Selecting previously unselected package python2.7.
    Preparing to unpack .../python2.7_2.7.6-8_amd64.deb ...
    Unpacking python2.7 (2.7.6-8) ...
    Selecting previously unselected package python-minimal.
    Preparing to unpack .../python-minimal_2.7.5-5ubuntu3_amd64.deb ...
    Unpacking python-minimal (2.7.5-5ubuntu3) ...
    Selecting previously unselected package python.
    Preparing to unpack .../python_2.7.5-5ubuntu3_amd64.deb ...
    Unpacking python (2.7.5-5ubuntu3) ...
    Selecting previously unselected package python-all.
    Preparing to unpack .../python-all_2.7.5-5ubuntu3_amd64.deb ...
    Unpacking python-all (2.7.5-5ubuntu3) ...
    Setting up libexpat1:amd64 (2.1.0-4ubuntu1) ...
    Setting up libffi6:amd64 (3.1~rc1+r3.0.13-12) ...
    Setting up libmagic1:amd64 (1:5.14-2ubuntu3) ...
    Setting up libssl1.0.0:amd64 (1.0.1f-1ubuntu2) ...
    Setting up libasprintf0c2:amd64 (0.18.3.1-1ubuntu2) ...
    Setting up libglib2.0-0:amd64 (2.40.0-2) ...
    No schema files found: doing nothing.
    Setting up libpipeline1:amd64 (1.3.0-1) ...
    Setting up libxml2:amd64 (2.9.1+dfsg1-3ubuntu4) ...
    Setting up groff-base (1.22.2-5) ...
    Setting up bsdmainutils (9.0.5ubuntu1) ...
    update-alternatives: using /usr/bin/bsd-write to provide /usr/bin/write (write) in auto mode
    update-alternatives: using /usr/bin/bsd-from to provide /usr/bin/from (from) in auto mode
    Setting up man-db (2.6.7.1-1) ...
    Building database of manual pages ...
    Setting up libcroco3:amd64 (0.6.8-2ubuntu1) ...
    Setting up libunistring0:amd64 (0.9.3-5ubuntu3) ...
    Setting up libpython2.7-minimal:amd64 (2.7.6-8) ...
    Setting up python2.7-minimal (2.7.6-8) ...
    Setting up file (1:5.14-2ubuntu3) ...
    Setting up mime-support (3.54ubuntu1) ...
    update-alternatives: using /usr/bin/see to provide /usr/bin/view (view) in auto mode
    Setting up gettext-base (0.18.3.1-1ubuntu2) ...
    Setting up gettext (0.18.3.1-1ubuntu2) ...
    Setting up intltool-debian (0.35.0+20060710.1) ...
    Setting up po-debconf (1.0.16+nmu2ubuntu1) ...
    Setting up dh-apparmor (2.8.95~2430-0ubuntu5) ...
    Setting up debhelper (9.20131227ubuntu1) ...
    Setting up libpython2.7-stdlib:amd64 (2.7.6-8) ...
    Setting up libpython-stdlib:amd64 (2.7.5-5ubuntu3) ...
    Setting up python2.7 (2.7.6-8) ...
    Setting up python-minimal (2.7.5-5ubuntu3) ...
    Setting up python (2.7.5-5ubuntu3) ...
    Setting up python-all (2.7.5-5ubuntu3) ...
    Processing triggers for libc-bin (2.19-0ubuntu6) ...
    Reading package lists...
    Building dependency tree...
    Reading state information...
    Reading extended state information...
    Initializing package states...
    Writing extended state information...
     -> Finished parsing the build-deps
    Reading package lists...
    Building dependency tree...
    Reading state information...
    The following extra packages will be installed:
      libfakeroot
    The following NEW packages will be installed:
      fakeroot libfakeroot
    debconf: delaying package configuration, since apt-utils is not installed
    0 upgraded, 2 newly installed, 0 to remove and 0 not upgraded.
    Need to get 0 B/80.4 kB of archives.
    After this operation, 415 kB of additional disk space will be used.
    Selecting previously unselected package libfakeroot:amd64.
    (Reading database ... 13945 files and directories currently installed.)
    Preparing to unpack .../libfakeroot_1.20-3ubuntu2_amd64.deb ...
    Unpacking libfakeroot:amd64 (1.20-3ubuntu2) ...
    Selecting previously unselected package fakeroot.
    Preparing to unpack .../fakeroot_1.20-3ubuntu2_amd64.deb ...
    Unpacking fakeroot (1.20-3ubuntu2) ...
    Processing triggers for man-db (2.6.7.1-1) ...
    Setting up libfakeroot:amd64 (1.20-3ubuntu2) ...
    Setting up fakeroot (1.20-3ubuntu2) ...
    update-alternatives: using /usr/bin/fakeroot-sysv to provide /usr/bin/fakeroot (fakeroot) in auto mode
    I: Copying back the cached apt archive contents
    I: Copying source file
    I: copying [../walinuxagent_1.3-0ubuntu1.dsc]
    I: copying [../walinuxagent_1.3-0ubuntu1.tar.gz]
    I: Extracting source
    dpkg-source: warning: extracting unsigned source package (walinuxagent_1.3-0ubuntu1.dsc)
    dpkg-source: info: extracting walinuxagent in walinuxagent-1.3
    dpkg-source: info: unpacking walinuxagent_1.3-0ubuntu1.tar.gz
    I: Building the package
    I: Running cd tmp/buildd/*/ && env PATH="/usr/sbin:/usr/bin:/sbin:/bin" dpkg-buildpackage -us -uc  -rfakeroot
    dpkg-buildpackage: source package walinuxagent
    dpkg-buildpackage: source version 1.3-0ubuntu1
    dpkg-buildpackage: source distribution raring
    dpkg-buildpackage: source changed by Ben Howard <ben.howard@ubuntu.com>
     dpkg-source --before-build walinuxagent-1.3
    dpkg-buildpackage: host architecture amd64
     fakeroot debian/rules clean
    dh clean --with python2
       dh_testdir
       dh_auto_clean
    Traceback (most recent call last):
      File "setup.py", line 26, in <module>
        import setuptools
    ImportError: No module named setuptools
    dh_auto_clean: python setup.py clean -a returned exit code 1
    make: *** [clean] Error 1
    dpkg-buildpackage: error: fakeroot debian/rules clean gave error exit status 2
    E: Failed autobuilding of package
    I: unmounting dev/pts filesystem
    I: unmounting run/shm filesystem
    I: unmounting proc filesystem
    I: cleaning the build env 
    I: removing directory /var/cache/pbuilder/build//40387 and its subdirectories

I don't know a workaround for this - I am guessing that the fakeroot environment is not getting setuptools installed, but I don't know how to fix this.

--- a/debian/control
+++ b/debian/control
@@ -3,7 +3,7 @@ Section: python
 Priority: extra
 Maintainer: Ben Howard <ben.howard@ubuntu.com>
 XSBC-Original-Maintainer: Microsoft Corporation <walinuxagent@microsoft.com>
-Build-Depends: debhelper (>= 8), python-all
+Build-Depends: debhelper (>= 8), python-all, python-setuptools
 Standards-Version: 3.9.3
 XS-Python-Version: all
 Homepage: http://go.microsoft.com/fwlink/?LinkId=250998

